src/tests/multihost/sssd/testlib/common/helper_functions.py

```diff
@@ -0,0 +1,50 @@
+import re
+from .expect import pexpect_ssh
+
+
+ds_instance_name = 'example1'
+
+
+def check_login(user, hostname, password):
+    """This function will check user login
+    user: Name of the user.
+    hostname: Name of the machine where user will login.
+    password: User password.
+    """
+    client = pexpect_ssh(hostname, user, password, debug=False).fast_login()
+    assert client
+
+
+def clear_only_domain_log(multihost):
+    """
+    This function will clear domain logs
+    """
+    client = multihost.client[0]
+    log_ssd = f'/var/log/sssd/sssd_{ds_instance_name}.log'
+    client.run_command(f'> {log_ssd}')
```

Comment:
I haven't tried it but, I would think you should be able to zero the logs with an empty string in put_file_contents like this:

`    client.put_file_contents(log_sssd, '')
`